src/containers/Balance/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { styled } from '@mui/material';
+import { formatUnits } from 'viem';
+import { useCustomTheme, useTokenList } from '~/hooks';
+
+export const Balance = () => {
+  const tokenList = useTokenList();
+  return (
+    <Card>
+      <ul>
+        {tokenList.map((t) => (
+          <li key={`${t.tokenData.chainId}-${t.tokenData.address}`}>
+            <p>
+              <b>{t.tokenData.name}</b>
+              <br />
+              Balance: {formatUnits(BigInt(t.balance), t.tokenData.decimals)}
+              <br />
```

Comment:
Consider using Typography component for better spacing and semantics

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/1/commits/63a34b63ee2e60025bd129994d870a51fff83d11